src/factories/workflow-specific/Restricted_PIM_Factory_v1.sol

```diff
@@ -0,0 +1,186 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity 0.8.23;
+
+import "forge-std/console.sol";
+
+// Internal Interfaces
+import {IOrchestratorFactory_v1} from
+    "src/factories/interfaces/IOrchestratorFactory_v1.sol";
+import {IOrchestrator_v1} from
+    "src/orchestrator/interfaces/IOrchestrator_v1.sol";
+import {IFM_BC_Bancor_Redeeming_VirtualSupply_v1} from
+    "@fm/bondingCurve/interfaces/IFM_BC_Bancor_Redeeming_VirtualSupply_v1.sol";
+import {IFM_BC_Restricted_Bancor_Redeeming_VirtualSupply_v1} from
+    "@fm/bondingCurve/interfaces/IFM_BC_Restricted_Bancor_Redeeming_VirtualSupply_v1.sol";
+import {IRestricted_PIM_Factory_v1} from
+    "src/factories/interfaces/IRestricted_PIM_Factory_v1.sol";
+import {IBondingCurveBase_v1} from
+    "@fm/bondingCurve/interfaces/IBondingCurveBase_v1.sol";
+import {IModule_v1} from "src/modules/base/IModule_v1.sol";
+
+// Internal Implementations
+import {FM_BC_Restricted_Bancor_Redeeming_VirtualSupply_v1} from
+    "@fm/bondingCurve/FM_BC_Restricted_Bancor_Redeeming_VirtualSupply_v1.sol";
+
+// External Interfaces
+import {IERC20} from "@oz/token/ERC20/IERC20.sol";
+
+// External Implementations
+import {ERC20Issuance_v1} from "src/external/token/ERC20Issuance_v1.sol";
+
+// External Dependencies
+import {ERC2771Context, Context} from "@oz/metatx/ERC2771Context.sol";
+
+contract Restricted_PIM_Factory_v1 is
```

Comment:
You added the Natspec headers to the interfaces, instead of to the contracts. We have them on the contracts, not the interfaces so far. 
Yesterday in the dev guidelines call we decided we want to have it in both actually (as I also see that having it in the interface makes a lot of sense). We will then (later) add the functionality to the CI bot to verify that both interface and contract have the same Natspec header (just so we don't forget to add it for one of them or if we change it but don't update the other one). Does that make sense?